01-what-is.asciidoc

```diff
@@ -2,4 +2,24 @@
 [[whatis_chapter]]
 == What is the Lightning Network?
 
-Initial paragraph
+Think of a thunderstorm, where electricity is ionizing channels of air to dissipate excess energy and neutralize voltage differences that have been building up during the storm. 
+A channel of ionized air forms when the voltage or potential difference is greater than the breakdown voltage (..kV) between the different regions in the cloud. 
+As the electric current makes its way through the fractal network of ionized air channels it chooses the path of least resistance to reach its destination, a path is formed between the charged up cloud and the ground of earth below. 
+Then a bright flash of light followed by a loud bang occurs as the current dissipated in the path of ionized air channels creates a hot plasma.
+The Lightning Network is named after this natural phenomenon, where the voltage differences are channel balances and the current is a stream of bitcoin of which ownership is transmitted through those channels.
+When lightning strikes the ground, the potential differences are settled at the cost of dissipating energy and making a loud bang.
+In the Lightning Network this happens when payment channels close, the final state of the channel is transmitted to the Bitcoin network with a fee for the miners to include it in the next block.
```

Comment:
I learned physics in my native language, therefore I'm having a really difficult time to follow the analogy, due to not being familiar with the terminology. I admit it's a badass analogy, but making readers stop reading from the very first paragraph may not be the best way to go.  
The counter argument count be is that, at this point everyone is paying full attention and whoever chews through this first paragraph will find the following parts easier.  
Anyhow, not against it, but I think it worth a discussion.

Reply:
I will make a nice visual diagram to go with it, so that the concept is clear even without having learned the physics jargon.

Reply:
Agree with Adam, you risk to lose 90% of readers at the first paragraph :) I have a PhD in physics but still I'm not sure what the point was.

Reply:
I like the analogy, but agree with some of the comments here that its too complex and risks losing readers straight away.   It may be better to start with language such as "lightning is a way to improve upon the bitcoin protocol with additional functionality that increases transaction throughput and reduces settlement time. "